routes/index.js

```diff
@@ -115,8 +178,9 @@ module.exports = function (passport) {
     res.redirect('/');
   });
 
+
   router.get('/userManagement', checkAuth, function (req, res) {
```

Comment:
There is a duplicate `router.get('/userManagement')`

Reply:
Hey @danieleynis, those are two different methods (one is get and one is post - used for search a specific user in search bar).

Reply:
@huynguyencs Ok, let's just merge what we have for now. I can show you how to squash you commits at tonights meeting and we can merge at the meeting. I'll hold off on approving until the meeting.